mail_room.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency "rake"
   gem.add_development_dependency "rspec"
-  gem.add_development_dependency "mocha"
+  gem.add_development_dependency "mocha", "~> 1.9.0"
```

Comment:
Thanks. I'd love to get off mocha and bourne. rspec should have sufficient functionality at this point in time to do it.